### ISSUE TITLE
behavior tests tripping llvm assert

### ISSUE THREAD

**Issue by andrewrk** [ziglang/zig#1875]
Interesting - the FreeBSD LLVM package has assertions enabled, so we get this nice message:

```
Assertion failed: (V[I]->getType() == T->getTypeAtIndex(I) && "Initializer for composite element doesn't match!"), function ConstantAggregate, file /wrkdirs/usr/ports/devel/llvm70/work/llvm-7.0.1.src/lib/IR/Constants.cpp, line 951.
```

I'm waiting for a debug build of LLVM to finish so that I can troubleshoot this. It should be reproducible on non-FreeBSD OS's like this:

```
bin/zig test ../test/behavior.zig --target-os freebsd --target-arch x86_64
```

I'm going to leave #1874 merged, since it appears it did not cause the issue; but rather exposed an existing issue.

**Comment by andrewrk** [comment]
This is actually tripping on all targets.

**Comment by andrewrk** [comment]
I believe this is fixed by 5864d92b4049e6d60a21a3f22220464808dd0518. I'll close this issue when the CI tests pass.

**Comment by andrewrk** [comment]
Looks to be fixed. The MacOS failure is an issue with homebrew.
